examples/basics/visuals/graph.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+This example demonstrates how to visualise a NetworkX graph using the
+GraphVisual.
+"""
+
+import sys
+
+import networkx as nx
```

Comment:
you'll probably need to add `networkx` as something that gets installed on `Travis` for this to pass the example build. Prefer a `conda` package if it's available, otherwise `pip` is fine.